src/containers/shared/components/TxDetails.tsx

```diff
@@ -384,6 +385,16 @@ const TxDetails = (props: Props) => {
     );
   }
 
+  function renderTicketCreate(): ReactElement {
+    const { t, instructions } = props;
+    const { ticketCount } = instructions;
+    return (
+      <div className="ticketCreate">
+        <span className="label">{t('ticket_count')}:</span> <span>{ticketCount}</span>
+      </div>
```

Comment:
not a big deal, but i find that when you have a literal space that you want to add to the markup it's more clear to either do:

```javascript
<span>{ }</span>
```

or

```javascript
<span>&nbsp;</span>
```